### ISSUE TITLE
BitmapFromHWND_Ex() is unnecessary.. you can use BitmapFromScreen()

### ISSUE THREAD

**Issue by mmikeww** [Millimeter-Git/AhkGdip_ImageSearchForChrome#1]
In the main GDIP library, there are two separate functions you can use, if you want to get the bitmap for a specific window

Gdip_BitmapFromHwnd() is the normal one, and it uses PrintWindow

Gdip_BitmapFromScreen() is the other. And it uses BitBlt just like your _Ex() version. To use it for a window instead of a screen, you simply pass the "hwnd:" string at the start. You can see the notes in the Gdip lib for that function.

Basically you can delete your whole BitmapFromHwnd_Ex() function, and just use the built-in BitmapFromScreen() instead
